src/app/home/pages/PlansPage/components/MigrationsTable.tsx

```diff
@@ -59,7 +59,9 @@ const MigrationsTable: React.FunctionComponent<IProps> = ({
   const rows: IRow[] = [];
   currentPageItems.forEach((migration) => {
     const { tableStatus } = migration;
-    const type = migration.spec.stage ? 'Stage' : 'Migration';
+    // Type should be rollback if spec.rollback=true, else use value in spec.stage
+    const stageOrMigrate = migration.spec.stage ? 'Stage' : 'Migration';
```

Comment:
Looks like we might be able to remove these?

Reply:
Resolved over slack